GenumKit/Parsers/StoryboardParser.swift

```diff
@@ -84,10 +84,11 @@ public final class StoryboardParser {
     storyboardsSegues[storyboardName] = delegate.segues
   }
 
-  public func parseDirectory(path: String) {
+  @available(OSX 10.10, *)
+  public func parseDirectory(path: String, osx: Bool = false) {
     if let dirEnum = NSFileManager.defaultManager().enumeratorAtPath(path) {
       while let subPath = dirEnum.nextObject() as? NSString {
-        if subPath.pathExtension == "storyboard" {
+        if subPath.pathExtension == "storyboard" && subPath.containsString("osx") == osx {
```

Comment:
wait, how is this still in the code? Didn't we decide that we shouldn't make that logic — that use the name of the file to decide if it's OS X or not — in the source of SwiftGen?

I guess since you now reorganized the fixtures directory into subdirectories to be able to separate iOS and OS X fixtures, this is no longer necessary, right?